src/shared/utils/exportData/getSubscales.ts

```diff
@@ -104,7 +104,7 @@ export const calcScores = <T>(
 
     let value: number | null = null;
 
-    if ('options' in typedOptions && typedOptions.options.length) {
+    if (typedOptions && 'options' in typedOptions && typedOptions.options.length) {
```

Comment:
Wouldn't a general

```ts
if (typedOptions) {
	// ...
}
```

wrapping around these two be a bit cleaner? (Avoiding checking two times for the same value)